cmake/BuildMariaDBConnector.cmake

```diff
@@ -9,7 +9,7 @@ set(MARIADB_CONNECTOR_C_REPO "https://github.com/MariaDB/mariadb-connector-c.git
   CACHE STRING "MariaDB Connector-C Git repository")
 
 # Release 2.3.3 (preliminary) of the Connector-C
-set(MARIADB_CONNECTOR_C_TAG "v3.0.2"
```

Comment:
https://jira.mariadb.org/browse/MDEV-14647 this bug will lead monitor crash so current need master version

Reply:
The bug is fixed but not yet released. Once the next Connector-C 3.0 version is released, this should be updated.

I suggest that this change is reverted and only updated once we know the tag.